command/secrets_tune_test.go

```diff
@@ -200,6 +204,18 @@ func TestSecretsTuneCommand_Run(t *testing.T) {
 			if exp := 3600; mountInfo.Config.MaxLeaseTTL != exp {
 				t.Errorf("expected %d to be %d", mountInfo.Config.MaxLeaseTTL, exp)
 			}
+			if exp := []string{"authorization", "www-authentication"}; !reflect.DeepEqual(exp, mountInfo.Config.PassthroughRequestHeaders) {
```

Comment:
It's fine to do it this way, but for future reference we often use `deep.Equal` instead: it returns a diff, which can allow for more helpful failure messages.

Reply:
Done